lib/middleman-simple-thumbnailer/extension.rb

```diff
@@ -37,12 +37,35 @@ def after_build(builder)
       def image_tag(path, options={})
         resize_to = options.delete(:resize_to)
         return super(path, options) unless resize_to
+        ext = app.extensions[:middleman_simple_thumbnailer]
 
-        image = MiddlemanSimpleThumbnailer::Image.new(path, resize_to, app)
         if app.development?
-          super("data:#{image.mime_type};base64,#{image.base64_data}", options)
+
+          # For better performance on local development
+          # (high bandwidth)
+          # simply change width in <img>-tag
+          
+          original_width  = options.delete(:width)
+          resized_width   = resize_to.split('x').first.to_i
+          original_height = options.delete(:height)
+          resized_height  = resize_to.split('x').last.to_i
+          
+          if !original_width.nil? && original_width <= resized_width
+            if original_height <= resized_height
+              options[:width] = original_width
```

Comment:
To be honest, this is my first too...

This is not really related to #13, but could be important.

Just a quick question: What is your use case for this extension ?
Do you have a bunch of images in a well-known location that need to be resized with the same size, or do you have a lot of different images with different resizing need each time ?

Thank you,

ymh